src/_guides/libraries/library-tour.md

```diff
@@ -1293,6 +1293,77 @@ await Future.wait([
 print('Done with all the long steps!');
 ```
 
+`Future.wait()` returns a future which will complete once all the provided
+futures have completed, either with their results, or with an error if any of
+the provided futures fail. 
+
+#### Handling errors for multiple futures
+
+You can also wait for parallel operations on an [iterable]({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/FutureIterable/wait.html)
+or [record]({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/FutureRecord2/wait.html)
+of futures.
+
+These extensions return a future with the resulting values of all the provided
+futures. Unlike `Future.wait`, if any future in the collection completes with
+an error, `wait` completes with a [`ParallelWaitError`][] that allows the caller
+to handle individual errors and dispose successful results if necessary.
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
```suggestion
These extensions return a future with the values of all provided futures.
This differs from `Future.wait`.
If any future in the collection returns an error, `wait` returns a [`ParallelWaitError`][].
The caller can then handle each error and drop successful results if needed.
```

Reply:
I went through your reviews bottom up on this section, but now that I'm up here, I realized that I could kind of rearrange the whole section to be more concise. I noticed I repeated what the new `wait` does 3 times (how it differs from `Future.wait`, then how it works for iterables, then how it works for records, which are all the same description). So I changed everything to now only mention how it works _once_ at the top, and then focus only on the differences:

> These extensions return a future with the resulting values of all provided
> futures. Unlike `Future.wait`, they also let you handle errors. 
> 
> If any future in the collection completes with an error, `wait` completes with a
> [`ParallelWaitError`][]. This allows the caller to handle individual errors and
> dispose successful results if necessary.
(NOTE: I think "completes" is different than "returns", so had to leave that in)
> 
> Use `wait` on an _iterable_ of futures when you _don't_ need the result values
> from each individual future:
> 
> ```dart
> // examples the same
> ```
> 
> Use `wait` on a _record_ of futures when you _do_ need the individual result values
> from each future. This also provides the additional benefit that the futures can be of different types:
>
> ```dart
> // examples the same
> ```
> (end section)